### PR TITLE
Add help section to admin (see owncast/owncast#541)

### PR DESCRIPTION
**Pull request by thilobillerbeck**
As discussed in https://github.com/owncast/owncast/issues/541, I implemented a help page with useful links for users looking for help.

### CODE REVIEW

**Comment by gabek** [comment]
This is awesome!

**Comment by gingervitis** [comment]
Hey cool! Quick comment, could you make the page a bit more responsive? Like maybe at a screen width < 1024, just put things in a single column?

**Comment by thilobillerbeck** [comment]
@gingervitis I added some basic responsive behavior to the elements. I think I might improve on that later, but it should work for now.

File: pages/help.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        {
+            icon: <ApiTwoTone style={{ fontSize: '24px' }}  />,
+            title: "I want to use the API",
+            content: (
+                <div>
+                    You can view the API documentation for either the 
+                    <a href="https://owncast.online/api/latest" target="_blank" rel="noopener noreferrer"> latest</a>
+                    or 
+                    <a href="https://owncast.online/api/development" target="_blank" rel="noopener noreferrer"> development</a>
+                    release.
```
Comment (jeyemwey) [inline pages/help.tsx:110]: This formatting removes some spaces:

image

Comment (thilobillerbeck) [reply pages/help.tsx]: strangely, my formatting worked in my chromium based browser. The latest commit should resolve it by using `&nbsp;`

**Comment by gabek** [comment]
Thank you for all of your work, this is a great addition to help our goal of bringing documentation closer to the user. Merging!

**Comment by graywolf336** [comment]
Major props for working on this and contributing it!!! 👏🏼

**Pull request merged**